manager/controllers/app/dataset/data_info.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package dataset
+
+import (
+	"errors"
+
+	"fybrik.io/fybrik/pkg/serde"
+
+	app "fybrik.io/fybrik/manager/apis/app/v1alpha1"
+	"fybrik.io/fybrik/manager/controllers/utils"
+	pb "fybrik.io/fybrik/pkg/connectors/protobuf"
+)
+
+// DataDetails is the asset metadata received from the catalog connector
+// This structure is in use by the manager and other components, such as policy manager and config policies evaluator
+type DataDetails struct {
+	// Name of the asset
+	Name string
+	// Interface is the protocol and format
+	Interface app.InterfaceDetails
+	// Geography is the geo-location of the asset
+	Geography string
```

Comment:
Isn't this part of the metadata?  Why is it called out separately?

Reply:
There was some technical issue with using grpc response structure as is. I don't remember all the details but 1. it should eventually go away as we get rid of the grpc structures and 2. it is not relevant for this PR - I just moved the structure to a new package because it was causing import loops.

Reply:
OK.  Please open an issue to address it though